src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -98,5 +98,5 @@
       "Size": 1904
     }
   },
-  "PackageSize": 3148309
+  "PackageSize": 3090965
```

Comment:
This looks like it solves the minor app size regression from:
* https://github.com/dotnet/android/issues/10198

The number we used to have was `3078677`, but that could have moved around slightly from other changes.